app/migrations/versions/6794c860baaf_.py

```diff
@@ -40,7 +40,7 @@ class CustomAttributeUserStorage(Base):
 def upgrade():
     conn = op.get_bind()
     session = Session(bind=conn)
-    op.create_index(op.f('ix_custom_attribute_user_storage_custom_attribute_id'), 'custom_attribute_user_storage', ['custom_attribute_id'], unique=False)
+    # op.create_index(op.f('ix_custom_attribute_user_storage_custom_attribute_id'), 'custom_attribute_user_storage', ['custom_attribute_id'], unique=False)
```

Comment:
having migration issues - think this is covered in another PR too, but just to stop breaking her